drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -231,59 +445,61 @@ static int sample_fetch(const struct device *dev, enum sensor_channel chan)
 
 	switch (chan) {
```

Comment:
Is this an actual requirement for sensor drivers? You know I wanted to avoid modifying the actual logic beyond supporting the new part number, but this bothered me incredibly since it breaks the shell command. Not sure why I did not fix it on my own, but I did now, thanks for pointing it out.